sessions/recap-project-3/rick-and-morty-app/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "rick-and-morty-starter",
```

Comment:
currently ghcd will use the package name as the folder name it creates

We should do one of those:
1. rename the package to `rick-and-morty-app`
2. or add another param to ghcd alá `ghcd <github-path> <foldername>` that would be consistent with how clone and other tools work

now that I think of it… why not both…

Reply:
```suggestion
  "name": "rick-and-morty-app",
```

Reply:
I like the idea of having a `ghcd` argument for a folder name. I put an issue into the backlog.

https://github.com/neuefische/web-curriculum-new-format/issues/243